code/go/0chain.net/smartcontract/storagesc/blobber.go

```diff
@@ -404,13 +420,16 @@ func (sc *StorageSmartContract) commitBlobberRead(t *transaction.Transaction,
 			"error fetching blobber object")
 	}
 
+	const CHUNK_SIZE = 64 * KB
```

Comment:
This is declared twice, here and in[ blober2_test.go](https://github.com/0chain/0chain/blob/staging/code/go/0chain.net/smartcontract/storagesc/blobber2_test.go#L25). I suggest merging the two.